### PR TITLE
Add community Java library links.

### PR DESCRIPTION
**Pull request by JohnLCaron**
Added links to the java library, as requested.

---
This change is [Reviewable](https://reviewable.io/reviews/microsoft/electionguard/48)

### CODE REVIEW

File: docs/guide/Core_Component.md [hunk 1]
```diff
@@ -13,6 +13,13 @@ _Includes: Ballot Encryption_
 
 :fontawesome-brands-github: Source - ✨ Coming Soon
 
+## Java
+
+_Includes: Complete port of python library_
```
Comment (keithrfung) [inline docs/guide/Core_Component.md:18]: You can include this line
`Includes: Key Generation, Ballot Encryption, Tallying, Decryption`

File: docs/guide/Core_Component.md [hunk 2]
```diff
@@ -13,6 +13,13 @@ _Includes: Ballot Encryption_
 
 :fontawesome-brands-github: Source - ✨ Coming Soon
 
+## Java
```
Comment (keithrfung) [inline docs/guide/Core_Component.md:16]: Can we actually make this:
```
## Community

### Java
```

Comment (keithrfung) [reply docs/guide/Core_Component.md:16]: This way it can be highlighted that it's done by the community.

**Review by keithrfung — CHANGES_REQUESTED** [review]
Minor changes since this is our first Community piece.

This is a nit but if you have time, can you add a little java image and link here:
https://github.com/microsoft/electionguard/blob/main/docs/Getting_Started.md

File: docs/guide/Core_Component.md (hunk 1 above)
Comment (JohnLCaron) [reply docs/guide/Core_Component.md:18]: done

File: docs/guide/Core_Component.md (hunk 2 above)
Comment (JohnLCaron) [reply docs/guide/Core_Component.md:16]: done

File: docs/guide/Core_Component.md (hunk 1 above)
Comment (JohnLCaron) [reply docs/guide/Core_Component.md:20]: Whats with ":fontawesome-brands-github:"? It doesnt seem to resolve when I look at the page on github??

Comment (keithrfung) [reply docs/guide/Core_Component.md:20]: We use github pages with a markdown theme. 

https://www.electionguard.vote/

Comment (keithrfung) [reply docs/guide/Core_Component.md:20]: https://www.electionguard.vote/guide/Core_Component/

**Review by keithrfung — CHANGES_REQUESTED** [review]
One more change. Lose the python port line. You can run `make install` and `make serve` locally if you want to see what it looks like. 

Screen Shot 2021-01-08 at 9 50 09 AM
Screen Shot 2021-01-08 at 9 50 52 AM

**Review by keithrfung — APPROVED** [review]

**Pull request merged**